server/plugin/api.go

```diff
@@ -289,8 +289,6 @@ func (p *Plugin) handleSetDateTimeDialog(w http.ResponseWriter, r *http.Request)
 		Placeholder: "YYYY-MM-DD",
 		HelpText:    "Please enter the date in the format YYYY-MM-DD. Example: 2001-11-04",
 		Optional:    false,
-		MinLength:   10,
```

Comment:
Does removing these checks show the error as "Invalid date"?

Reply:
We are getting this `Please enter a valid date`.